# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/squashfs-tools/hotfix-pthread.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

w/ musl:

In file included from compressor.c:29:
print_pager.h:33:25: error: unknown type name 'pid_t'
   33 | extern void wait_to_die(pid_t process);
      |                         ^~~~~
print_pager.h:34:25: error: unknown type name 'pid_t'
   34 | extern FILE *exec_pager(pid_t *process);
      |                         ^~~~~

--- ./squashfs-tools/print_pager.h.vanilla	2025-07-24 21:29:11.186025627 +0200
+++ ./squashfs-tools/print_pager.h	2025-07-24 21:29:35.443154051 +0200
@@ -23,6 +23,8 @@
  * print_pager.h
  */
 
+#include <pthread.h>
+
 #define TRUE 1
 #define FALSE 0
 
